### ISSUE TITLE
install help uses wrong names

### ISSUE THREAD

**Issue by adhamhashibon** [simphony/simphony-osp#846]
both the user docs and the readme are a bit hard to follow. it is not clear that one should clone and either run the install from within the `simphony-osp` folder like so `pip install .` or run 'pip install ./simphony-osp` from out side the folder!

**Comment by kysrpex** [comment]
I believe you may be getting confused because probably last time you tried SimPhoNy it was not [available on PyPI](https://pypi.org/project/simphony-osp/).

The [README](https://github.com/simphony/simphony-osp/blob/v4.0.0/README.md) says:

> SimPhoNy is available on PyPI, so it can be installed using pip

The [documentation says](https://simphony.readthedocs.io/en/v4.0.0/introduction/installation.html#installation):

> SimPhoNy is [available on PyPI](https://pypi.org/project/simphony-osp/), so it can be installed using the [pip package manager](https://pip.pypa.io/en/stable/).

And both statements are followed by the command:

```shell
pip install simphony-osp
```

This command will **pull the packge `simphony-osp` from [PyPI](https://pypi.org/project/simphony-osp/)**. In this case, **you do not need to clone anything!!**

If you want to **install the package from source**, then the [README](https://github.com/simphony/simphony-osp/blob/v4.0.0/README.md) says:

> Detailed installation instructions can be found [here](https://simphony.readthedocs.io/en/v4.0.0/introduction/installation.html).

And the [documentation says](https://simphony.readthedocs.io/en/v4.0.0/introduction/installation.html#developers):

> If you are a developer or an advanced user, you might be interested in installing SimPhoNy from source.

> `git clone https://github.com/simphony/simphony-osp.git`
> `pip install ./simphony-osp`

**Comment by kysrpex** [comment]
@adhamhashibon Did the previous message make things more clear? Can the issue be closed?
